reps/2022-03-10-plasma_alternative.md

```diff
@@ -0,0 +1,185 @@
+
```

Comment:
I think besides the explicitly defined `PlasmaClientInterface`, we also need to find all the hidden/implicit assumptions/dependencies Ray has on the object store. One thing I can think of is the scheduler, currently Ray treats object store memory as a resource and it will try to avoid scheduling tasks on node where object store is almost full. Does this still work with third-party object store especially when other systems will add objects to it that's out of the awareness of Ray.

Reply:
Please refer to the summary below.